attendance/resources/azureface.py

```diff
@@ -0,0 +1,39 @@
+import os
+import logging
+
+from azure.cognitiveservices.vision.face import FaceClient
+from flask import current_app
+from msrest.authentication import CognitiveServicesCredentials
+
+from resources.vault import obtain_data
+
+log = logging.getLogger('root')
+
+
+def compare_faces_azure(targetId, sourceId):
```

Comment:
No one calls this method. Need to define environment variables: AZURE_FACE_ENABLED and AWS_REKOG_ENABLED. Call the service only if the environment variable is set.